frontend/packages/console-app/src/components/guided-tours/utils/explore-serverless-tour.ts

```diff
@@ -0,0 +1,86 @@
+export const exploreServerlessTour = {
+  apiVersion: 'console.openshift.io/v1',
+  kind: 'GuidedTours',
+  metadata: {
+    name: 'explore-serverless',
+  },
+  spec: {
+    version: 2.6,
```

Comment:
With `version` I was aiming to match the version of the console, to accuratelly reflect the workflow in particular release of OpenShift's web console.
Assuming that the `2.6` is the version of the operator? in the context of Guided Tours, Im not sure if it makes since the Guided Tour should cover not just operators(which are versioned), but also various worflows(which arenot versioned). 
From the operator perspective we are showing just one version of operator in the console right, so its version should not matter?

Reply:
I don't think version would matter in any of the current use cases.

Reply:
However, I am concerned that the `kind` might change given that now `GuidedTours` are called `QuickStarts` @christianvogt